tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -676,6 +678,7 @@ object TastyFormat {
     case TYPELAMBDAtype => "TYPELAMBDAtype"
     case LAMBDAtpt => "LAMBDAtpt"
     case MATCHtype => "MATCHtype"
+    case MATCHCASEtype => "MATCHCASEtype"
```

Comment:
I think TastyPrinter will need a case too.

Reply:
I don't see why?

Reply:
Ah maybe not, I see we don't need cases for all type nodes in the printer.